api/db/migrate/20201115193346_add_catalog_year_to_plans.rb

```diff
@@ -0,0 +1,5 @@
+class AddCatalogYearToPlans < ActiveRecord::Migration[6.0]
+  def change
+    add_column :plans, :catalog_year, :integer, default: 2018
```

Comment:
Wondering if this should default to the current year when a user makes a plan. In this case we don't need a default here